src/common/avatars/BaseAvatar.js

```diff
@@ -5,14 +5,16 @@ import styled from 'styled-components/native';
 import CachedImage from '../CachedImage';
 
 const StyledCachedImage = styled(CachedImage)`
-  border-radius: ${({ size }: Object): number => size / 2};
+  border-radius: ${({ rounded, size }: Object): number =>
```

Comment:
You can just use `border-radius: 50%`

I think you should have two classes. One for styling and one for adding the border radius instead of adding logic for generating these styles.